### DISCUSSION TITLE
Duda informe punto 1 - Representación de estructuras

### DISCUSSION THREAD

**Discussion by SFCYang** [IIC2133-PUC/2024-1#8] [Tarea 0] [answered]
Entiendo que para el punto 1 del informe nos piden identificar las principales estructuras y sus variables (members), pero a qué se refiere con "como se pueden representar"?

Gracias de antemano!

**Comment by sachondo7** [comment, marked as answer]
No es necesario que te compliques tanto, es solamente explicar por ejemplo para que elementos de la tarea usarías un tipo de estructura de datos u otra y  además explicar porque sería ideal usar esa y no otra. 
El informe no tiene pauta como tal, la idea de este es que sea breve y lo que se buscar evaluar es el modelamiento que están proponiendo y que este tenga sentido. 
Saludos!
